ee-security/pom.xml

```diff
@@ -29,6 +29,16 @@
         <relativePath/>
     </parent>
 
+    <properties>
```

Comment:
This should follow https://docs.google.com/document/d/1GFnZrnlwRkwC0HfziRws2UUWR99KdaQayA7Ei12SGN4/edit#heading=h.q4f7r5l2mqax

```
<properties>
      <!-- the version for the Server -->
      <version.server>30.0.0.Beta1</version.server>
      <!-- The versions for BOMs, Packs and Plugins -->
      <version.bom.ee>${version.server}</version.bom.ee>
      <version.pack.cloud>5.0.0.Beta1</version.pack.cloud>
      <version.plugin.wildfly>4.2.0.Final</version.plugin.wildfly>
  </properties>

```

Reply:
Fixed!

Reply:
Fixed! 
Thank you for your review!